src/temp/stories/VegetationChange.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import SatelliteIcon from '../../../public/assets/images/icons/SatelliteIcon';
+import type { Meta, StoryObj } from '@storybook/react';
+import SingleTab from '../ProjectViewTabs/SingleTab';
+
+const meta: Meta<typeof SingleTab> = {
+  component: SingleTab,
+};
+
+export default meta;
+type Story = StoryObj<typeof SingleTab>;
+
+export const TimeTravelSelected: Story = {
+  args: {
+    icon: <SatelliteIcon color={'#fff'} />,
+    title: 'Time Travel',
+    isSelected: true,
+  },
+};
+
+export const TimeTravelUnSelected: Story = {
```

Comment:
Text changes
1. Use a more generic name like `TabSelected` / `TabUnselected`, or even `Selected` / `Unselected` (describing the state only, as we already have the component name from the file)?
2. Storybook separates words based on Pascal Case, so `UnSelected` should be written as `Unselected` as it is one word.